squad-server/plugins/discord-server-status.js

```diff
@@ -99,6 +99,10 @@ export default class DiscordServerStatus extends DiscordBaseMessageUpdater {
     // Set footer.
     embed.setFooter(COPYRIGHT_MESSAGE);
 
+    // Clamp the ratio between 0 and 1 to avoid tinygradient errors.
+    const ratio = this.server.a2sPlayerCount / (this.server.publicSlots + this.server.reserveSlots);
+    const clampedRatio = Math.min(1, Math.max(0, ratio));
+    
```

Comment:
Please run `yarn lint`. There is a space it is axing here.

Reply:
Yes, I apologize. I made the original changes using the github editor without running lint.